package/skelcd-control-suse-manager-proxy.spec

```diff
@@ -61,7 +62,7 @@ Provides:       system-installation() = SUSE-Manager-Proxy
 URL:            https://github.com/yast/skelcd-control-suse-manager-proxy
 AutoReqProv:    off
 # IMPORTANT: This needs to be 4.3.0 as it is the SUSE Manager version!
-Version:        4.3.0
+Version:        4.3.1
```

Comment:
I see latest suse manager version still 4.3.3, so 4.3 should be fine here.

Reply:
> I  see latest suse manager version still 4.3.3


**4.3.3**? where did you see it?

Reply:
it is even 4.3.5 in HEAD https://build.suse.de/package/view_file/Devel:Galaxy:Manager:Head/spacewalk-backend/spacewalk-backend.spec?expand=1